### ISSUE TITLE
"R.swift/rswift: No such file or directory" v5.0.0.alpha.2

### ISSUE THREAD

**Issue by Ethenyl** [mac-cain13/R.swift#464]
I'm trying to make `R.Swift` work with **Swift 4.2** and **XCode 10.1** and I'm having a strange issue.

Using the tagged version `v5.0.0.alpha.2` is resulting in a missing `rswift` script in the Pods output directory.

Hence the build is failing every single time, sole solution for now is to enforce the compilation to **Swift 4.0** and fork `R.swift.library` in a local repo. Since `v4.0.0` can't be used with an App compiled with **Swift 4.2** because of a compilation issue with `NibResourceType.instantiate(withOwner ownerOrNil:options optionsOrNil:`.

I'm using the latest stable version of **CocoaPods**, the **v1.5.3**. I've tried to use the tag directly and even using the shortcut to the tag, no joy for now.

**Comment by frankatstyleshoots** [comment]
I have the exact same issue, with the added bonus of all "return instantiate(withOwner: ownerOrNil, options: optionsOrNil)[0] as? UIKit.UIView" statements failing with "Use of unresolved identifier 'instantiate'".

**Comment by mac-cain13** [comment]
Are you both using `pod 'R.swift', '5.0.0.alpha.3'` in your Podfile? This is what we're using in our projects and is working just fine.

**Comment by frankatstyleshoots** [comment]
I just tried the 5.0.0.alpha.3 release and still get this: "<Path to project root>/Pods/R.swift/rswift: No such file or directory". The instantiate thing is gone though.
I'm using Xcode 10.1 (10B61) and Cocoapods 1.5.3. Same result with Cocoapods 1.6.0.beta.2. I tried both the legacy and the new build system, removing derived data and reinstalling the pods, but the result is the same. :confused:  I'll see if I can distill a li'l test project to replicate the error...
